### PR TITLE
[Merged by Bors] - feat: port MeasureTheory.Function.AEMeasurableSequence

### PR DESCRIPTION
**Pull request by int-y1**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Function/AEMeasurableSequence.lean [hunk 1]
```diff
@@ -9,19 +9,20 @@ Authors: Rémy Degenne
 ! if you have ported upstream changes.
 -/
 import Mathlib.MeasureTheory.MeasurableSpace
+import Mathlib.MeasureTheory.Measure.MeasureSpaceDef
```
Comment (RemyDegenne) [inline Mathlib/MeasureTheory/Function/AEMeasurableSequence.lean:12]: That import is new? Why is it needed?

**Review by RemyDegenne — COMMENTED** [review]
The new naming convention improves the lemma names in this file a lot :)
I have a question about an import you apparently added. Otherwise LGTM.

File: Mathlib/MeasureTheory/Function/AEMeasurableSequence.lean (hunk 1 above)
Comment (Parcly-Taxel) [reply Mathlib/MeasureTheory/Function/AEMeasurableSequence.lean:12]: In mathlib3 this file imports `measure_theory.measurable_space` which imports the **tactic file** `measure_theory.tactic` which imports `measure_theory.measurable_space_def` which contains the declaration `measure_theory.measure` required for _this_ file. In mathlib4 there is no `MeasureTheory.Tactic` file, so the existing import chain as directly transported to mathlib4 doesn't have `MeasureTheory.Measure` available. So the extra import is needed.

Comment (RemyDegenne) [reply Mathlib/MeasureTheory/Function/AEMeasurableSequence.lean:12]: Thanks for the explanation!

**Pull request closed without merging**